config/config.go

```diff
@@ -171,7 +171,9 @@ var (
 			RepeatDecayStep:       1,
 		},
 		Dispatcher: Dispatcher{
-			EventChanSize: 10000,
+			ActionChanSize:    1000,
+			BlockChanSize:     1000,
+			BlockSyncChanSize: 10,
```

Comment:
why use 10 so small compared to 1000? consider sync chan is unicast only (vs block chan is broadcast + unicast), i would say use 200~400 for BlockSyncChanSize

my concern is that this would potentially slowdown sync speed of full-node